packages/at_secondary_server/lib/src/connection/outbound/outbound_client.dart

```diff
@@ -224,6 +230,7 @@ class OutboundClient {
         return false;
       }
     } on ConnectionInvalidException {
+      logger.severe('Invalid connection: ${toString()}');
```

Comment:
I guess you are intending this would be `${e.toString()}`? In which case you need to do `on ConnectionInvalidException catch (e) {`

Reply:
My intention was to log the OutboundClient.toString(). In your opinion, do you think this provides any useful insight while debugging stuff ?